gradle/project.gradle.kts

```diff
@@ -112,26 +114,25 @@ dependencies {
         exclude("xpp3", "xpp3_min")
     }
 
-    // Image
+    // Images
```

Comment:
In my opinion `Image` fits better that `Images`, particularly considering the name `Audio` above. Consistency is good. hence `misc` should also begin with an upper case. It doesn't really matter, but it does make a (small) difference. I do not suggest to fix this here, I'm just commenting.

Ultimately, I do not approve of comments like this and I'd rather see a code block, or named code block to segregate the groups in the future.

Reply:
Why a code block? But I do understand your nitpicking, I'm usually very similar 😉

Reply:
This may be subjective, but an empty line in a code should be a greater granularity of grouping than here. The empty line + comment style is not exactly ideal (even if I practiced this a lot in the past). Comments are almost always a bad idea.

For example here `run("Image") { compile... }` blocks would be much better, because it would allow retain the meaning of the empty line separators while enforcing each compile section to have a clear uniform structure. Comments are much more likely to get out of hand (abundant, missing, arbitrary naming) that rigorous run() blocks.

This is more of a suggestion, but I think it has a solid reasoning. What do you think?

Reply:
I think `"Audio" { ... }` style should also be possible because of kotling gradle DSL

Reply:
Hmm I feel like this unecessarily adding code that doesn't have a function. Comments aren't code, and since this is just visual aid, I feel like they are right here.

Reply:
I see, but you have no problem using Kotlin as a build file, which is a domain of stuff like xml. One would argue (and I debated this at work already with sbt for Scala) that language (typed one at that) has no place in the build file.

The programming language offers advantages for more than just a code. You can see it with TODO() method in the Kotlin API. That too has been a comment once.

In my opinion comments are not only out of fashion, but also have no place in the code whatsoever. I've seen it over and over. Dangling comments, comments that are out of context, misleading or downright incorrect or unintelligible to anyone (including the author himself). Comments are incredibly fragile and become obsolete very quick (faster than Javadoc). Comments ended up proliferating the problem they were supposed to solve.

There should only be code (and annotations) in the code file. This also helps refactoring, reuse and searchability as the tools can leverage the language support. Ingraining the meaning into the constructs of the language is too advantageous to ignore.

I'm marking this resolved of course, but I'll probably return to this at some point.

Reply:
I can see your point, but this is a case whether they won't become obsolete or dangling, since the buildscript is relatively static. Also, there is no inherent advantage of using any other construct in here in terms of IDE usability and stuff like that.